abn/k8sclient/k8sclient.go

```diff
@@ -76,3 +114,8 @@ func (c *KubeClient) Typed() kubernetes.Interface {
 func (c *KubeClient) Dynamic() dynamic.Interface {
 	return c.dynamicClient
 }
+
+// GVR determines the GroupVersionResource of an object
+func (c *KubeClient) GVR(o *unstructured.Unstructured) (*schema.GroupVersionResource, error) {
```

Comment:
I see another function called `getKind()`. Should this be renamed to `getGVR()` to match then?

Reply:
This has been removed so question no longer relevant.